definitions/divorce/json/CaseEvent.json

```diff
@@ -1070,6 +1074,7 @@
     "PostConditionState": "SOTAgreementPayAndSubmitRequired",
     "CallBackURLAboutToSubmitEvent": "${CCD_DEF_COS_URL}/solicitor-create",
     "RetriesTimeoutURLAboutToSubmitEvent": "120,120",
+    "CallBackURLSubmittedEvent": "${CCD_DEF_COS_URL}/solicitor-created",
```

Comment:
I'm not sure what this does but it's not related to DA. If it's to do with draft petition being created and deleted, it should already be in prod

Reply:
We don't want this at all! This shouldn't be in master either...

Reply:
@alexwakeman mind creating a PR to remove it in master? I'll then update this branch with master

Reply:
yup doing it now